### ISSUE TITLE
About DAOUQuery condition

### ISSUE THREAD

**Issue by churow** [phpMv/ubiquity#159]
If I had a DAOUQuery condition like this:
```
$emps = DAO::uGetAll(Emp::class, "... AND (jobs3.job_date IS NULL OR jobs3.job_date = '') ...", [..., 'jobs3.*']);
```
The final query looked like "... AND (_jobs3_.job_date IS NULL OR _job_e5fbcfeddaf0f6_.job_date = '') ...", where the first one was not replaced into table alias.

It was because of _(^|\s)_ part of uParseExpression().

vendor/phpmv/ubiquity/src/Ubiquity/orm/traits/DAOUQueries.php
```
protected static function uParseExpression():
	// ...
	$ucondition = preg_replace ( '/(^|\s)' . $expression . '/', " {$jSQL["alias"]}." . $expressionArray [0], $ucondition );
	// ...
```

I had to have a space between the opening parenthesis and the member name, jobs3.

It needs to be noted somewhere in the user guide.

**Comment by jcheron** [comment]
Hi @churow 
`uParseExpression` could also make a correct replacement, including the possibility that a parenthesis can be placed int the condition without a space.
I would rather qualify your discovery as a bug.

**Comment by jcheron** [comment]
Available in 2.3.13
